assets/src/edit-story/app/media/utils/useProcessVideo.js

```diff
@@ -82,6 +83,36 @@ function useProcessVideo({
     [updateMedia]
   );
 
+  const updateElementDimensions = useCallback(
```

Comment:
Is that a 1:1 copy from `useUploadWithPreview`?

We could de-dupe this by extracting it into a shared hook.

Reply:
Some of it is copied by not all sadly :(

Reply:
It is a copy. I will make a shared hook.

Reply:
Moved to shared hook.